os/arch/arm/Kconfig

```diff
@@ -45,6 +45,10 @@ config ARCH_CHIP
 	default "tiva"      if ARCH_CHIP_LM
 	default "bcm4390x"  if ARCH_CHIP_BCM4390X
 
+config ARCH_HAVE_PM
```

Comment:
Here are ARM related config. PM is not approprietary. Let's move this to Kconfig.chip or Kconfig.board

Reply:
moved it to Kconfig.chip